### DISCUSSION TITLE
plugins 不支持18.20

### DISCUSSION THREAD

**Discussion by sibo-git** [umijs/umi#11959] [Q&A]
├─┬ @umijs/plugins 4.0.89
│ ├─┬ @ahooksjs/use-request 2.8.15
│ │ └── ✕ unmet peer react@"^16.8.0 || ^17.0.0": found 18.2.0
│ ├─┬ react-intl 3.12.1
│ │ └── ✕ unmet peer react@^16.3.0: found 18.2.0
│ └─┬ dva-loading 3.0.24
│   └── ✕ unmet peer dva-core@"^1.1.0 || ^1.5.0-0 || ^1.6.0-0": found 2.0.4 in @umijs/plugins
└─┬ dva 2.5.0-beta.2
  ├── ✕ unmet peer react@"15.x || ^16.0.0-0": found 18.2.0
  ├── ✕ unmet peer react-dom@"15.x || ^16.0.0-0": found 18.2.0
  └─┬ react-redux 5.1.2
    └── ✕ unmet peer react@"^0.14.0 || ^15.0.0-0 || ^16.0.0-0": found 18.2.0

**Comment by fz6m** [comment]
告警并不会影响实际使用，由于社区中的第三方包并不是都会跟进版本的变化，所以目前存在大量的 peer deps 不匹配现象，无需注意。

另外不推荐继续使用 dva 和 use request 了，这是 umi 3 旧时的兼容存在，现代数据流更推荐使用 [swr](https://github.com/vercel/swr) 、[react-query](https://umijs.org/docs/max/react-query) 管理请求，[jotai](https://github.com/pmndrs/jotai) 、[valtio](https://umijs.org/docs/max/valtio) 管理全局数据。

如正在使用 pnpm ，由于 pnpm 迭代频繁，请更新 pnpm 至最新版本，防止出现不必要的问题。
